pkg/resourceinterpreter/interpreter.go

```diff
@@ -68,12 +79,12 @@ type customResourceInterpreterImpl struct {
 func (i *customResourceInterpreterImpl) Start(ctx context.Context) (err error) {
 	klog.Infof("Starting custom resource interpreter.")
 
-	i.customizedInterpreter, err = customizedinterpreter.NewCustomizedInterpreter(i.kubeconfig, i.informer)
+	i.customizedInterpreter, err = customizedinterpreter.NewCustomizedInterpreter("", i.informer)
```

Comment:
Why not remove the parameter from `NewCustomizedInterpreter`, since it doesn't need it anymore?

Reply:
Yes, currently we do not set this value, it is always empty, previously analyzed to pass in an empty string is OK.